doc/symfony-cache-configuration.rst

```diff
@@ -143,6 +146,79 @@ one of ``client_ips`` or ``client_matcher``*.
 
   **default**: ``PURGE``
 
+
+Purge tags (cache invalidation using tags)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+    You need at least Symfony 3.4 to use this feature!
```

Comment:
i think this is not strictly correct. you need symfony/lock and symfony/cache at 3.4, but afaik you can use symfony 3.3 together with those components, or am i wrong?

we should tell the composer require line for those two components, because they are optional dependencies.